components/SitesList.js

```diff
@@ -0,0 +1,14 @@
+import styled from "styled-components";
+import { allSites } from "../lib/sites";
+
+export default function SitesList() {
+  const sites = allSites.map((site) => {
+    return (
+      <li>
+        <mark>{site.pos[0] + " | " + site.pos[1]}</mark>
```

Comment:
If you want to apply a backgrond color for all Items, please to that with CSS. The <mark> is only for Highligting.

Reply:
Have te problem with block element when using CSS. But will try again... As I am not sure if I am going to keep the list optic, I might not want to use so much time on that for now ...